public/components/SelectDataTable.vue

```diff
@@ -1,79 +1,77 @@
 <template>
-  <fixed-header-table ref="fixedHeaderTable">
-    <b-table
-      bordered
-      hover
-      small
-      :items="items"
-      :fields="tableFields"
-      @sort-changed="onSortChanged"
```

Comment:
I guess we don't need to sort the table with the new <b-table> component?

Reply:
Again, this was used to kick off fixed table state management.